File: PHP/Baju.php
<?php

class Baju extends Aksesoris {
    private $forwho;
    private $size;
    private $brand;

    public function __construct($id = -1, $name = "", $category = "", $price = 0, $stock = 0, $image = "", $type = "", $material = "", $color = "", $forwho = "", $size = "", $brand = "") {
        parent::__construct($id, $name, $category, $price, $stock, $image, $type, $material, $color);
        $this->forwho = $forwho;
        $this->size = $size;
        $this->brand = $brand;
    }
    
    public function setBajuData($id, $name, $category, $price, $stock, $image, $type, $material, $color, $forwho, $size, $brand) {
        parent::setAksesorisData($id, $name, $category, $price, $stock, $image, $type, $material, $color);
        $this->forwho = $forwho;
        $this->size = $size;
        $this->brand = $brand;
    }
        
    public function getForwho() {
        return $this->forwho;
    }

    public function getSize() {
        return $this->size;
    }

    public function getBrand() {
        return $this->brand;
    }
}

File: Python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    
    __forwho = ""
    __size = 0
    __brand = ""

    def __init__(self,id = -1, name = "", category = "", price = 0, stock = 0, type="", material="", color="", forwho="", size=0, brand=""):
        super().__init__(id, name, category, price, stock, type, material, color)
        self.__forwho = forwho
        self.__size = size
        self.__brand = brand

    def set_baju_data(self, id, name, category, price, stock, type, material, color, forwho, size, brand):
        super().set_aksesoris_data(id, name, category, price, stock, type, material, color)
        self.__forwho = forwho
        self.__size = size
        self.__brand = brand

    def get_forwho(self):
        return self.__forwho
    
    def get_size(self):
        return self.__size
    
    def get_brand(self):
        return self.__brand